components/brave_wallet/common/common_util.cc

```diff
@@ -29,4 +29,18 @@ bool IsAllowed(PrefService* prefs) {
   return !IsDisabledByPolicy(prefs);
 }
 
+std::vector<uint8_t> Leb128Encode(size_t value) {
+  value |= 0;
```

Comment:
`value |= 0` What's the purpose of this line?

Reply:
Removed